https://issues.redhat.com/browse/ACM-13627 for 2.12 (#6834)
open-cluster-management/rhacm-docs: install/install_overview.adoc

FIPS notice: If you do not specify your own ciphers in `spec.ingress.sslCiphers`, then the `multiclusterhub-operator` provides a default list of ciphers. If you upgrade and want FIPS compliance, remove the following two ciphers from the `multiclusterhub` resource: `ECDHE-ECDSA-CHACHA20-POLY1305` and `ECDHE-RSA-CHACHA20-POLY1305`.

> The documentation references the earliest supported {ocp} versions, unless the component in the documentation is created and tested with only a specific version of {ocp-short}. For full support information, see the link:https://access.redhat.com/articles/7073065[Support matrix] and the link:https://access.redhat.com/support/policy/updates/advanced-cluster-management[Lifecycle and update policies for {acm}].

Author: ```suggestion
The documentation references the earliest supported {ocp-short} version, unless a specific component or function is introduced and tested only on a more recent version of {ocp-short}.

For full support information, see the link:https://access.redhat.com/articles/7073065[Support matrix] and the link:https://access.redhat.com/support/policy/updates/advanced-cluster-management[Lifecycle and update policies for {acm}].
```
